hera_sim/simulate.py

```diff
@@ -321,12 +326,24 @@ def _check_compatibility(self):
         if self.data.phase_type != "drift":
             raise CompatibilityException("The phase_type of the data must be 'drift'.")
 
-    def _iterate_antpair_pols(self):
+    def _iterate_antpair_pols(self, **kwargs):
         """
         Iterate through antenna pairs and polarizations in the data object
         """
-
+        # Only operate on certain baselines
+        use_baselines = None
+        if 'use_baselines' in kwargs.keys():
+            use_baselines = kwargs['use_baselines']
+        if use_baselines is None:
+            use_baselines = self._use_baselines
```

Comment:
I think this can be simply `use_baselines = kwargs.get('use_baselines', None) or self._use_baselines`